src/joining_type.rs

```diff
@@ -0,0 +1,73 @@
+use std::collections::{BTreeMap, BTreeSet};
+
+use ucd_parse::{self, ArabicShaping};
+
+use args::ArgMatches;
+use error::{Error, Result};
```

Comment:
It looks like there is an unused import here?

Also, I would just squash this back into the original commit. :-)

Reply:
Ahh dang it, sorry about that. Fixed now.